### ISSUE TITLE
Missing ./alias-tool?

### ISSUE THREAD

**Issue by jonpugh** [consolidation/site-alias#65]
I was poking around and just learned about `alias-tool.sh`...

Not seeing this on the repo.

I'm thinking this tool might be used for a new "drush launcher"?

**Comment by jonpugh** [comment]
I found it. Was removed when php8 tests were implemented:
https://github.com/consolidation/site-alias/commit/6355893e0800d2b3a5a248e4f6acc25dc20a3cd8

**Comment by greg-1-anderson** [comment]
Yeah, it wasn't really being used for anything and was hard to maintain, so I just removed it. Perhaps the old code might be of some use to you. Be aware that it was never more than sample code, though.
